### PR TITLE
Author statement & hide funders section for PLOS

### LINKED ISSUE #219: Manuscript updates required for resubmission
The PLOS editorial team sent the following:
> Thank you for submitting your paper, 'Open collaborative writing with Manubot' to PLOS Computational Biology! We are looking forward to passing your manuscript on for the editors' evaluation; however, after an initial check of your files, we would first ask you to address the following points:

> -Please provide an Author Summary. This should appear in your manuscript between the Abstract and the Introduction, and should be 150–200 words long. The aim should be to make your findings accessible to a wide audience that includes both scientists and non-scientists. Guidelines and sample summaries can be found here: http://journals.plos.org/ploscompbiol/s/submission-guidelines#loc-author-summary

> -Please remove the funding information from your article file and ensure that it is entered completely and correctly into the Financial Disclosure section of the online submission form.

> -Please add this sentence to your funding Information: “The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.” If this sentence does not apply, please add appropriate wording to indicate what role the funders played. You can access and modify this field by going to the “Additional Information” tab when editing the submission.

We can write the summary and add it after the abstract.  Our abstract already describes what Manubot does.  Maybe the summary should focus more on the challenges of collaborative writing and significance of Manubot?

I can easily update the funding information in the submission system.  What do we want to do with the funding statement in the manuscript?  We could remove it entirely or remove it and add back the funding information in `00.front-matter.md`.

@dhimmel do you have time to work on the author summary or should I take a pass?  We can release `v3.1` after making these changes.

### PR DESCRIPTION
**Pull request by dhimmel**
WIP

Closes https://github.com/greenelab/meta-review/issues/219

> Please provide an Author Summary. This should appear in your manuscript between the Abstract and the Introduction, and should be 150–200 words long. The aim should be to make your findings accessible to a wide audience that includes both scientists and non-scientists. Guidelines and sample summaries can be found here: https://journals.plos.org/ploscompbiol/s/submission-guidelines#loc-author-summary

### CODE REVIEW

**Comment by dhimmel** [comment]
Let's iterate on the version in https://github.com/greenelab/meta-review/pull/220/commits/074358913da0abec49d5a88319e0503c5bd1cb57. @agitter / @slochower feel free to review now.

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: Given space constraints, we can remove `such as this one you are reading`

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: Do we need to introduce the idea that typically a fairly small number of authors wrote the manuscript in this setting here?  It may be sufficiently covered in the next paragraph.

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: To avoid the `But now` `But how` sentence structure, we could phrase this as something like:
> Dispersed teams of online contributors require new tools to jointly prepare manuscripts to describe their work

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: We could add a concrete number here or below.  `fail to scale beyond tens of authors` or whatever we think is reasonable.

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: We can explain GitHub to make it more accessible: `on the version control platform GitHub`.  That exact phrase may not be sufficiently accessible though.

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: Start with `Manubot also addresses` to cut words

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: `automates many tasks` is shorter

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: We could cut this sentence to save space.  It is an important feature, but a very similar sentence exists in the abstract.

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: `permalink` is correct, but `permanent URL` or `stable URL` would be more accessible.  Do we want to advertise these as "permanent" given potential stability issues with GitHub Pages?

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: Can you please clarify how they are like preprints?  We could refocus this to say:
> Manubot manuscripts can be continuously updated, where readers have ample opportunity to not only provide peer review but also to contribute improvements.

**Review by agitter — CHANGES_REQUESTED** [review]
I definitely like the approach and message.  I have detailed comments on how we may refine some of the phrasing.  I did not yet try to fit it in the 200 word limit.

I'd also like to add a sentence stating that these Manubot features also make it attractive for traditional private workflows.  We don't want readers to think they should avoid Manubot if they aren't doing open collaborative writing.

File: content/01.abstract.md
Comment (slochower) [reply content/01.abstract.md]: Agree, it's not necessary.

File: content/01.abstract.md
Comment (slochower) [reply content/01.abstract.md]: A bit less jargon:

> Therefore, we created a system for writing collaborative manuscripts incorporating automatic versioning, called Manubot.

File: content/01.abstract.md
Comment (slochower) [reply content/01.abstract.md]: Agree.

File: content/01.abstract.md
Comment (slochower) [reply content/01.abstract.md]: I like the rephrasing. `where readers` sounds funny to me, but I'm not sure how I'd rephrase it. Maybe I'd say `whereby` to mean "...manuscripts can be updated by readers..." but that sounds archaic, so I'm okay with `where` in your version.

File: content/01.abstract.md
Comment (agitter) [reply content/01.abstract.md]: I agree we don't need to name GitHub here.  It appears in the abstract.

File: content/01.abstract.md
Comment (agitter) [reply content/01.abstract.md]: We could use
> continuously updated, meaning readers...

or
> continuously updated.  Readers...

I'm a fan of short, simple sentences in the summary paragraph.

Comment (dhimmel) [reply content/01.abstract.md]: Take a look at the last sentence in ddd5222b987a04a9740d577130024e3cee7a2964 (still long) and suggest any improvements you see fit. I want to tie this in with preprints, since I think that is an important topic these days. Primarily, how manubot takes preprints to the next level by:

1. making them more customizeable, automated, and versioned
2. making them more open from early stages when feedback is most actionable

Comment (slochower) [reply content/01.abstract.md]: I think you could expand and split into two sentences. Something like...

> As such, Manubot produces an ecosystem of living preprint. 
> Readers of in-progress and complete manuscripts have ample opportunity to not only provide public peer review but also to contribute improvements, before and after publication.

What do you think?

Comment (agitter) [reply content/01.abstract.md]: This idea is a good closing.  Can we push more on how it takes preprints to the next level, echoing @dhimmel's points and phrasing above?  That may not be implied strongly enough by referring to "living" preprints?  Maybe

> As such, Manubot extends the benefits of preprints.
Manuscript readers have ample opportunity to not only provide public peer review but also to contribute improvements, before and after publication.

The latest commits resolved everything else except the word count, which I didn't check.

Comment (dhimmel) [reply content/01.abstract.md]: Regarding word count, we are probably good (right around 200).

@agitter I like your phrasing. What do you think about keeping your second sentence, but replacing your first sentence with:

+ As such, Manubot is a suitable foundation for next-generation preprints.
+ As such, Manubot is a suitable foundation for the next generation of preprints.

No strong opinions, just wanted to see if this was appealing. In some other places we use the "next-generation manuscripts" tagline, so it could be good branding.

Comment (agitter) [reply content/01.abstract.md]: I prefer the first `next-generation preprints` phrasing.   It's a bold statement, but I'm okay with that.  We want to present a strong vision for the future.

Comment (dhimmel) [reply content/01.abstract.md]: Okay I think we should be good as of https://github.com/greenelab/meta-review/pull/220/commits/79f872040d86d94b481457ea872c515453584843. Let me know if I missed anything.

**Review by slochower — APPROVED** [review]

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: How about?
> Therefore, we created a system called Manubot for writing manuscripts based on collaborative version control.

**Review by agitter — APPROVED** [review]
This looks great.  I have one more optional suggestion.  I'm approving to indicate you can merge when ready regardless of whether you accept or reject this change.

After this is merged, should I tag as `v3.1` and resubmit?  I was planning only updating the manuscript PDF, not the diff or response letter.

File: content/01.abstract.md
Comment (dhimmel) [reply content/01.abstract.md]: Done in https://github.com/greenelab/meta-review/pull/220/commits/0cfaf7794c4abf6e1b6bf9efc6dbe21712973369

**Comment by dhimmel** [comment]
> After this is merged, should I tag as v3.1 and resubmit? I was planning only updating the manuscript PDF, not the diff or response letter.

I agree with those decisions.

**Pull request merged**
